kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -258,11 +297,71 @@ private void initialize() {
                 }
             }
         });
-
         // Setup Properties
         setupProperties();
     }
 
+    /**     *
```

Comment:
Can you add comments describing method? Can you remove the asterisk?

Reply:
Done.